guide/additional-info/async-await.md

```diff
@@ -35,11 +35,28 @@ deleteMessages(5).then(value => {
 });
 ```
 
-In this scenario the `deleteMessages` function returns a Promise. The `.then()` function will trigger if the Promise was resolved and the `.catch()` function if the Promise was rejected. But with our function we resolve the Promise after 2 seconds with the String "Everything is fine", so the `.catch()` function will never be executed.
+In this scenario the `deleteMessages` function returns a Promise. The `.then()` method will trigger if the Promise was resolved and the `.catch()` method if the Promise was rejected. But with our function we resolve the Promise after 2 seconds with the String "Everything is fine", so the `.catch()` method will never be executed.
 
 ## How to implement async/await
 
-After knowing how Promises works and what they are for, let's look at an example in which we handle multiple Promises. Let's say you wanted to react to a message in a certain order with letters (regional indicators). For this example, I will take the basic template for a discord.js bot with some ES6 adjustments.
+### Theory information
+The following Information are important to know before starting working with async/await. You can only use the await keyword inside a function what is marked as async (you call the async keyword before the function name, function keyword or the parameter when you use an arrow function) a simple example would be 
+```js
+async function markedAsAsync() {
+	// code
+}
+``` 
+or 
+```js 
+async () => {
+	// code
+}
+```
+also an important thing to know is that a function marked as async will always return a Promise. In addition to this if you return something inside a async function this value will be the resolved value and if you throw an error inside an async function it will reject this Promise with the error object. 
+
+
+### Execution with discord.js code
+After knowing how Promises works and what they are for aswell as about the theory information, let's look at an example in which we handle multiple Promises. Let's say you wanted to react to a message in a certain order with letters (regional indicators). For this example, I will take the basic template for a discord.js bot with some ES6 adjustments.
```

Comment:
- `aswell` -> `as well`
- same here, `information` in `theory information` feels a bit redundant
- should be `we will handle` (or `we'll handle`)
- `you want to react`
- `to react with letters (regional indicators) in a certain order`, feels nicer this way
- `I will` -> `you will`. (For examples second person, for general stuff third person.)